src/app/views/query-runner/request/permissions/Permission.tsx

```diff
@@ -172,43 +174,47 @@ export class Permission extends Component<IPermissionProps, IPermissionState> {
       }
     ];
 
+    const currentPermType = permissionModeType === DISPLAY_DELEGATED_PERMISSIONS;
+
```

Comment:
extra line

Reply:
I think this whitespace is appropriate for clarity.